### ISSUE TITLE
Any library for generating Spec

### ISSUE THREAD

**Issue by vijay-15372** [bazaarvoice/jolt#1184]
Is there any library available for generating JOLT Spec by providing the input and output JSON

**Comment by hveiras** [comment]
I don't think so, if that is possible without any other input then JOLT would not be necessary.
